pkg/auditlog/storage/v2/sql/adminauditlog/insert_admin_audit_log_v2.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO admin_audit_log (
+    id,
+    timestamp,
+    entity_type,
+    entity_id,
+    type,
+    event,
+    editor,
+    options
+) VALUES
```

Comment:
Let's refactor here so as not to use strings.Builder.

https://github.com/bucketeer-io/bucketeer/blob/main/pkg/account/storage/v2/sql/account_v2/insert_account_v2.sql#L1-L11